hazelcast-client/src/test/java/com/hazelcast/client/connectionstrategy/ConfiguredBehaviourTest.java

```diff
@@ -161,6 +172,7 @@ public void testReconnectModeASYNCSingleMemberInitiallyOffline() {
         HazelcastInstance hazelcastInstance = hazelcastFactory.newHazelcastInstance();
 
         clientConfig.getConnectionStrategyConfig().setReconnectMode(ASYNC);
+        clientConfig.getNetworkConfig().setConnectionAttemptLimit(Integer.MAX_VALUE);
```

Comment:
default value is 20 which means one minute timeout, I think it should be enough. Why do we set it to MAX